reference/api/overview.mdx

```diff
@@ -15,7 +15,7 @@ If you are new to Meilisearch, check out the [getting started](/learn/self_hoste
 
 ## OpenAPI
 
-You can get the [Meilisearch OpenAPI specifications here](https://github.com/meilisearch/open-api/blob/main/open-api.json).
+Meilisearch OpenAPI specifications (`meilisearch-openapi.json`) are attached to the [latest release of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/latest)
```

Comment:
FYI, I cannot link the file with `latest` in it -> https://github.com/meilisearch/meilisearch/releases/download/latest/meilisearch-openapi.json does not exists but https://github.com/meilisearch/meilisearch/releases/download/v1.17.1/meilisearch-openapi.json yes

Don't worry, for future automation on your side, it will not be a blocker, and we will be able to go around that